.github/workflows/check-advisories.yml

```diff
@@ -67,7 +67,7 @@ jobs:
             ID=$(basename "$FILE" .md)
             YEAR=$(echo "$ID" | cut -d - -f 2)
             mkdir -p $DATA_DIR/$YEAR
-            docker run --rm -v ${{ github.workspace }}:/advisories haskell/hsec-tools:latest /bin/hsec-tools osv "advisories/$FILE" > $DATA_DIR/$YEAR/$ID.json
+            docker run --rm -v ${{ github.workspace }}:/advisories haskell/hsec-tools:latest /bin/hsec-tools osv "repo/$FILE" > $DATA_DIR/$YEAR/$ID.json
```

Comment:
Don't we also need to change:

    -v ${{ github.workspace }}:/advisories

into:

    -v ${{ github.workspace }}:/repo

?